pom.xml

```diff
@@ -251,16 +251,23 @@
         </pluginManagement>
         <plugins>
             <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>cobertura-maven-plugin</artifactId>
-                <version>2.7</version>
-                <configuration>
-                    <formats>
-                        <format>html</format>
-                        <format>xml</format>
-                    </formats>
-                    <check/>
-                </configuration>
+                <groupId>org.jacoco</groupId>
```

Comment:
What's motivating the change? Is it related with the new way we want to run tests?

Reply:
Just discovered that cobertura maven plugin doesn't work properly with Java 8 (it's a bug that's been fixed for over 6 months but no release has been made). Most classes weren't being properly instrumented. Will replicate on the other repo.